src/LightingThread.cpp

```diff
@@ -516,9 +516,9 @@ void cLightingThread::CalcLightStep(
 {
 	UNUSED(a_IsSeedIn);
 	int NumSeedsOut = 0;
-	for (int i = 0; i < a_NumSeedsIn; i++)
+	for (size_t i = 0; i < static_cast<UInt32>(a_NumSeedsIn); i++)
```

Comment:
Use a temporary to remove the cast from each iteration (consider compilers / optimizers dumb)